src/components/Content/Home/HomeNav.tsx

```diff
@@ -1,123 +1,140 @@
-import { useState } from 'react'
-import { Dialog } from '@headlessui/react'
-import { Bars3Icon, XMarkIcon } from '@heroicons/react/24/outline'
-import  { useRouter } from 'next/router'
+import { useState } from "react";
+import { Dialog } from "@headlessui/react";
+import { Bars3Icon, XMarkIcon } from "@heroicons/react/24/outline";
+import { useRouter } from "next/router";
+import EastIcon from "@mui/icons-material/East";
 
 const navigation = [
```

Comment:
Suggestion: 
- instead of this, create a menu file (menu-items.ts of sort) so that when we need to update the links, we don't have to edit this file.